src/utils/config.ts

```diff
@@ -8,3 +8,5 @@ export const updaterConfig = {
   prefixTargetsWithHash: true,
   fetchTimeout: 100000, // milliseconds
 };
+
+export type Config = typeof updaterConfig;
```

Comment:
Let's rename `updateConfig` to `defaultConfig` since that's really what it is here.